demos/serverless/deploy.js

```diff
@@ -14,7 +14,7 @@ let enableTerminationProtection = false;
 let disablePrintingLogs = false;
 let chimeEndpoint = 'https://service.chime.aws.amazon.com';
 let chimeSDKMeetingsEndpoint = 'https://service.chime.aws.amazon.com';
-let chimeServicePrincipal = 'chime.amazonaws.com'
+let chimeServicePrincipal = 'mediapipelines.chime.amazonaws.com'
```

Comment:
Since this is media capture specific, can you rename this to chimeMediaCaptureServicePrincipal?